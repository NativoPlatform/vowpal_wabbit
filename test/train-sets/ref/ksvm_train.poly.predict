0
0.473179
0.043690
-0.221670
-0.383732
-0.461511
-0.153729
-0.233516
-0.484661
-0.562680
-0.326803
-0.487334
-0.415228
-0.517041
-0.590047
-0.328346
-0.191346
-0.024942
-0.519667
-0.282460
-0.566126
-0.204371
-0.134682
-0.219253
-0.157097
-0.315441
-0.506414
-0.659454
-0.261309
-0.108485
-0.490028
-0.167154
-0.397195
-0.266554
-0.407762
-0.286250
-0.431123
-0.240170
-0.319760
0.022751
-0.218293
-0.213734
-0.136208
-0.180990
-0.503167
-0.500598
-0.217634
-0.379842
-0.563389
-0.239617
-0.466050
-0.146295
0.008048
-0.174817
-0.132580
0.268461
-0.434388
-0.388533
-0.326374
-0.258337
-0.283590
-0.214389
0.103717
-0.616891
-0.218163
-0.116976
-0.175719
-0.110462
-0.585619
-0.298554
0.121925
0.067770
-0.219866
-0.437347
0.070643
0.002358
0.016176
-0.476153
-0.422687
-0.491685
-0.413959
-0.397858
-0.074233
-0.146782
-0.548549
-0.517684
-0.578876
0.075124
-0.004372
-0.524310
-0.402121
-0.175256
0.469811
-0.090564
0.315776
-0.409856
-0.381941
-0.097856
-0.350084
-0.235004
0.190451
-0.013100
-0.224065
0.040808
0.098082
0.007847
-0.317861
-0.274434
-0.372463
-0.138715
0.059225
0.099925
-0.174656
-0.436427
0.262518
-0.329150
-0.541911
0.079437
0.041551
-0.073014
0.043746
-0.154095
-0.052820
-0.259249
0.257015
0.336779
0.018050
0.344441
-0.349550
-0.187810
-0.049003
0.060413
0.070710
-0.175932
-0.033401
-0.408841
-0.065510
0.194689
0.891185
-0.146082
-0.439825
-0.522749
0.540468
0.890272
-0.230741
0.355666
-0.146295
0.765217
-0.743175
-0.109186
0.390741
0.092797
0.602887
0.111306
-0.385751
0.228808
0.623311
-0.028624
0.683376
0.071449
0.879100
-0.369276
-0.441046
0.203602
-0.114941
-0.153761
0.673723
-0.278977
-0.320803
0.487848
0.673978
-0.445086
-0.276026
0.200299
-0.476804
0.037242
-0.611362
0.705766
0.198773
0.522145
0.178717
0.186741
0.133779
-0.421139
-0.263317
-0.102413
0.204075
-0.398194
-0.352398
0.148491
0.540868
-0.151906
-0.417982
-0.343180
-0.401740
-0.001010
0.122079
-0.228896
-0.047296
-0.081329
-0.231545
0.177496
0.208077
-0.335930
-0.416465
-0.478392
0.500990
0.055120
0.000377
-0.111690
-0.194374
-0.611665
-0.508835
0.063233
1.135761
-0.637312
-0.456452
-0.250123
-0.273071
-0.189305
0.110615
-0.532250
-0.393052
-0.383491
0.121386
-0.072401
1.305149
0.217297
-0.368431
-0.601360
-0.649882
-0.621278
-0.687175
-0.001620
0.338299
-0.511985
-0.543143
0.206058
0.906753
0.122334
0.250082
-0.135272
-0.762486
0.473154
-0.080186
0.450464
0.516060
-0.311740
-0.571440
0.862587
