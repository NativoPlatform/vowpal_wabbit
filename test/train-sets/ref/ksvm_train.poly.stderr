using l2 regularization = 1
predictions = ksvm_train.poly.predict
Lambda = 1
Kernel = poly
degree = 2
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.236589 1.473179            2            2.0  -1.0000   0.4732      103
1.073800 0.911010            4            4.0  -1.0000  -0.2217      134
0.998217 0.922633            8            8.0  -1.0000  -0.2335      145
0.952234 0.906251           16           16.0   1.0000  -0.3283       23
0.946637 0.941040           32           32.0  -1.0000  -0.1672       31
0.938738 0.930838           64           64.0  -1.0000  -0.6169       60
0.927005 0.915273          128          128.0   1.0000   0.3444      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.821767
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 248
Number of kernel evaluations = 142029 Number of cache queries = 229854
Total loss = 205.441864
Done freeing model
Done freeing kernel params
Done with finish 
